### PR TITLE
KAA-1139: QA Review fixes

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @Polinaa

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-ESP8266/index.md [hunk 1]
```diff
@@ -358,7 +358,7 @@ which will be flashed to 0x00000 and 0x40000 flash addresses respectively.
 
         esptool.py elf2image build/kaa_demo
 
-Now these binaries can be written to Flash memory. But first, ensure that you have enabled flash mode on ESP8266 chip (see [here](#connecting-esp8266))
+Now these binaries can be written to Flash memory. But first, ensure that you have enabled flash mode on ESP8266 chip (see [here](#connecting-esp8266)).
 If everything done correctly, invoke this command:
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-ESP8266/index.md]: is done

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-ESP8266/index.md [hunk 2]
```diff
@@ -348,7 +348,7 @@ To invoke CMake, proceed as follows:
             -DKAA_MAX_LOG_LEVEL=3
         make
 
-For detailed description of available options refer to [Kaa C SDK page]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C) page.
+For detailed description of available options refer to [Kaa C SDK page]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C).
 
 ## Flashing
 Once the application has been built, you'll get a `kaa_demo` ELF executable in `build` directory.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-ESP8266/index.md]: in the `build`

**Comment by Polinaa** [comment]
Change `Fash mode` to `Flash mode` [in 31 line](https://github.com/kaaproject/kaa/pull/829/files#diff-3b08c50908b89eff26391cb0f807c488R31)

**Comment by RostakaGmfun** [comment]
@rasendubi

**Comment by Polinaa** [comment]
LGTM

**Pull request merged**